FFMpegCore/FFMpeg/Pipes/PipeHelpers.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using System;
+using System.IO;
 using System.Runtime.InteropServices;
 
 namespace FFMpegCore.Pipes
 {
     static class PipeHelpers
     {
-        public static string GetUnqiuePipeName() => $"FFMpegCore_{Guid.NewGuid()}";
+        public static string GetUnqiuePipeName() => $"FFMpegCore_{Guid.NewGuid().ToString("N").Substring(0, 5)}";
```

Comment:
How is uniqueness guarenteed with only 5 chars?

Reply:
Uniqueness is not guaranteed even with 32 or 36+ characters as well.
Considering this is a media processing library, the temporary files should be disposed of after each run and I don't see how a single disk can be used for highly concurrent media processing that can overlap with 5-character randomness.
Even if that happens, any implementer of a media processing solution should always have a fail-safe and retry logic, regardless of this.
So, I see generating longer names that breaks this library running on Mac does not provide any value.
But if you truly want to provide uniqueness, you can re-call this function to re-generate another name if the path already exists. Even in that case, you need a good locking mechanism.

Reply:
True, but the chance of it being unique is higher the more characters. Anyway, I agree that it probably won't cause any problems, and as the current solution does cause problems on macos I'm fine with this change